### PR TITLE
Update README.md

### LINKED ISSUE #1: CV copy
@Karyum has requested that I review his CV and proof the copy.

### PR DESCRIPTION
**Pull request by rebeccaradding**
Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
-# Mario Saliba - Full Stack Web Developer
+# CV - Mario Saliba
 
-I'm a young full stack developer who is looking for an opportunity to gain new skills. Just after school i joined a highly-selective sixteen week coding bootcamp called [Founders & Coders](http://www.foundersandcoders.com/), I recently graduated and started mentoring and already working on projects.
+I'm an early-career full stack developer who is looking for opportunities to gain new skills by working on challenging freelance projects. I recently graduated from [Founders & Coders Nazareth](http://www.foundersandcoders.com/), a highly selective sixteen week coding bootcamp, and am currently mentoring and working on freelance projects. I am a passionate learner and problem solver, eager to explore new languages and frameworks and learn with like-minded individuals.
```
Comment (Karyum) [inline README.md:3]: :+1:

File: README.md [hunk 2]
```diff
@@ -35,12 +36,26 @@ Nazareth Nuggets is a platform for people to discover new things about Nazareth,
...
+Park and Sleep, which helps tourists navigate unconventional cities (like Nazareth), was built using React library + Redux. 
 
+I used the following technologies:
 - React
 - Express
 - MongoDB
 - Webpack + Babel + Axios
 
-# Contact
+## EXPERIENCE
```
Comment (Karyum) [inline README.md]: Don't the projects also count as experience ?

Comment (rebeccaradding) [reply README.md]: @Karyum you're right, the word "EDUCATION" would be more appropriate.

Comment (Karyum) [inline README.md]: @rebeccaradding just changed can this be merged or is there other stuff to add ?

Comment (Karyum) [reply README.md]: @rebeccaradding changed it can it be merged now or is there anything else to add?

**Comment by rebeccaradding** [comment]
@Karyum happy for you to merge and we can see how it looks now!

**Pull request merged**